exercises/arrays/countEvens/countEvens.js

```diff
@@ -12,14 +12,37 @@
  * @returns {number} The count of even integers in the array
  */
 function countEvens(array) {
-  // This is your job. :)
+  for(i=0;i<array.length;i++)
+  {
+    if(array[i]%2 != 0)
+    {
+      array.splice(i,1)
```

Comment:
Hmmm. You're overthinking this. Do the index card exercise version of this and write it in a straightforward way.

You almost *never* want to use `array.splice` — it's expensive. It also modifies the input array, which is likely not what you want.